docs/matchbox-server.md

```diff
@@ -130,3 +138,18 @@ This mode will load the [FHIR Cross-Version Mapping Pack](https://build.fhir.org
 which contains _StructureMaps_ for all FHIR Core resources.
 Matchbox will also force the right version on the FHIR Core _StructureDefinitions_, to allow their use by the 
 _StructureMaps_.
+
+
+## LLM support {: #llm-support}
+Adding `llm` configurations will allow the server to make API calls to the specified LLM and add an analysis of the validation results to the operation outcome. This provides the user with AI generated instructions on how to fix errors in the validated FHIR resource.
+
+This feature requires a provider, model and API key to be defined in the applications configuration.
+
+Supported LLMs:
+
+| Provider                                            | Recommended Model           | Supported Models                                                                                               |
+|-----------------------------------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------|
+| [OpenAI](https://openai.com/index/openai-api/) (`openai`)     | `gpt-4o-mini`               | `gpt-3.5-turbo`, `gpt-4`, `gpt-4o`, `gpt-4o-mini`                                                              |
+| [Anthropic](https://www.anthropic.com/api) (`anthropic`)| `claude-3-5-sonnet-20241022`         | `claude-3-5-sonnet-20241022`, `claude-3-5-haiku-20241022`, `claude-3-sonnet-20240229`, `claude-3-opus-20240229`|
+
+To use this feature, `analyzeOutcomeWithAI` must be set to `true` by the user in the validation settings.
```

Comment:
analyzeOutcomeWithAI should be also listed in the configuration list with a default entry of false.